js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -323,18 +326,24 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
         <Button intent="none" onClick={() => setOpen(false)}>
           Cancel
         </Button>
-        <Button
-          intent="primary"
-          onClick={onLaunch}
-          disabled={selected.length === 0}
-          loading={launching}
-        >
-          {launching
-            ? 'Launching...'
-            : selected.length !== 1
-            ? `Launch ${selected.length}-Run Backfill`
-            : `Launch 1 Run`}
-        </Button>
+        {selected.length !== 1 && !canLaunchPartitionBackfill.enabled ? (
+          <Tooltip content={canLaunchPartitionBackfill.disabledReason}>
+            <Button disabled>{`Launch ${selected.length}-Run Backfill`}</Button>
+          </Tooltip>
+        ) : (
+          <Button
+            intent="primary"
+            onClick={onLaunch}
+            disabled={selected.length === 0}
+            loading={launching}
+          >
+            {launching
+              ? 'Launching...'
+              : selected.length !== 1
+              ? `Launch ${selected.length}-Run Backfill`
+              : `Launch 1 Run`}
+          </Button>
```

Comment:
Could possibly use a `<Tooltip canShow>` here to simplify things.

Reply:
Ahh forgot we had this! Going to make a seprate linear task to move some of these callsites to canShow! Looks like there are a bunch I could switch over